scss/_base_button.scss

```diff
@@ -49,20 +49,14 @@
       }
     }
 
-    &.is-dense {
-      margin-bottom: $spv-nudge-compensation;
-      padding-bottom: calc(#{$spv-nudge - $sp-unit * 0.5} - 1px);
-      padding-top: calc(#{$spv-nudge - $sp-unit * 0.5} - 1px);
-    }
-
     &.is-small {
       font-size: #{map-get($font-sizes, small)}rem;
       line-height: map-get($line-heights, small);
       margin: 0 0 $input-margin-bottom - $sp-unit 0;
       padding: calc(#{map-get($nudges, nudge--small)} - 1px) $sph-inner--small;
     }
 
-    &.is-small.is-dense {
+    &.is-dense {
       margin-bottom: $spv-nudge-compensation;
       padding-bottom: calc(#{$spv-nudge - $sp-unit * 0.5} - 1px);
```

Comment:
spv-nudge is the nudge on defualt text size. on the small text size it is different (`map-get($nudges, nudge--small)`). I have a feeling the problem with the existing code is that the two have been wrong, it's not a simple case of duplication.

Reply:
Ok, but if we want to fix it, we should do it separately, this was just removing duplication.
Changing this may affect other things, and we don't want that in this PR.

Reply:
jjust want to be careful about removing selectors that most probably should exist.

Reply:
@lyubomir-popov ah, fair point

@sowasred2012 Maybe let's not do it now, seems like Lyubo may be right. While these sections (`.is-dense` and `.is-small.is-dense`) are the same now, they probably shouldn't (because `is-small` uses different paddings (values of nudges).
So let's keep it as it was for now, and maybe add separate issue to check what padding bottom/top should `is-small.is-dense` have.